src/qibolab/platform/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .load import create_platform
+from .platform import Platform, unroll_sequences
+
+__all__ = ["Platform", "create_platform", "unroll_sequences"]
```

Comment:
This is probably trying to reproduce the previous behavior, however in principle we don't need to necessarily expose `unroll_sequences` externally. I think it is only used by the `Platform` internally.